docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Get and set recurrence in an Outlook add-in
+description: Get and set various recurrence properties of an item in an Outlook add-in.
+author: elizs
+ms.topic: article
+ms.technology: office-add-ins
+ms.date: 11/28/2018
+ms.author: elizs
+---
+
+# Get and set recurrence
+
+Imagine that you need to schedule a meeting with your team to discuss the current status of a project. It turns out that you need the status repeatedly until the project is completed. Instead of scheduling a meeting each time, you can configure the meeting to include how often you want it to occur. Then let's say that subsequently one of the meetings is at a time where you or one of the key attendees can't make it. You can then update that particular instance or occurrence, or cancel it altogether, without affecting the remainder of the series. Note that you can also set up a recurrence for your repeat appointments.
+
+**Table 1. Available item states**
+
+|State|Editable?|Viewable?|
+|---|---|---|
+|Appointment organizer - compose series|Yes ([`setAsync`][setAsync link])|Yes ([`getAsync`][getAsync link])|
+|Appointment organizer - compose instance|No (`setAsync` returns an error)|Yes ([`getAsync`][getAsync link])|
+|Appointment attendee - read series|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+|Appointment attendee - read instance|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+|Meeting request - read series|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+|Meeting request - read instance|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+
+## Available recurrence patterns
+
+To configure the recurrence pattern, you need to combine the recurrence type and its applicable recurrence properties (if any).
+
+**Table 2. Recurrence types and their applicable properties**
+
+|[Recurrence type](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetype)|Valid [recurrence properties](/javascript/api/outlook_1_7/office.recurrenceproperties)|Usage|
+|---|---|---|
+|`daily`|- [`interval`][interval link]|An appointment occurs every *interval* days. Example: An appointment occurs every **_2_** days.|
+|`weekday`||An appointment occurs every weekday.|
+|`monthly`|- [`interval`][interval link]<br/>- [`dayOfMonth`][dayOfMonth link]<br/>- [`dayOfWeek`][dayOfWeek link]<br/>- [`weekNumber`][weekNumber link]|1. An appointment occurs on day *dayOfMonth* every *interval* months. Example: An appointment occurs on day **_5_** every **_4_** months.<br/>2. An appointment occurs on the *weekNumber* *dayOfWeek* every *interval* months. Example: An appointment occurs on the **_third_** **_Thursday_** every **_2_** months.|
```

Comment:
On seeing the numbered list my first thought was these related to the listed properties in order. But I think these are just 2 examples of usage. I wouldn't number them since that implies an order (do one first, then another).